keyboards/wilba_tech/rama_works_m50_ax/rama_works_m50_ax.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2021 Jason Williams (Wilba)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+#include "via.h"
```

Comment:
```suggestion
```

Reply:
This is needed for `FN_MO13` and `FN_MO23`

Reply:
`quantum.h` already includes `via.h`.

Reply:
It only conditionally includes `via.h` when `VIA_ENABLE` is defined, and since the default keymap uses the tri-layer keycodes, it was included explicitly for that. However, this made me realize that the handling for the tri-layer keycodes would not be complied into non-VIA builds, so I've fixed that.